builtInFunctions/logsAndEvents.go

```diff
@@ -79,18 +79,29 @@ func newEntryForESDT(identifier, tokenID []byte, nonce uint64, value *big.Int, a
 	return logEntry
 }
 
-func extractTokenIdentifierAndNonceESDTWipe(args []byte) ([]byte, uint64) {
+func extractTokenIdentifierAndNonceESDTWipe(esdtPrefix []byte, args []byte) ([]byte, uint64) {
```

Comment:
Regarding previous comment, I think you should actually extract prefix from `args`

Reply:
`esdtPrefix` comes from protocol, `args`(or tokenIdentifier) is coming from user and he can send wrong identifier. 
I need to validate args based on something from protocol